pkg/testutils/manifest.go

```diff
@@ -0,0 +1,93 @@
+package testutils
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	declarative "github.com/kyma-project/lifecycle-manager/internal/declarative/v2"
+	"github.com/kyma-project/lifecycle-manager/pkg/module/common"
+	"github.com/kyma-project/lifecycle-manager/pkg/testutils/builder"
+	compdesc2 "github.com/open-component-model/ocm/pkg/contexts/ocm/compdesc/versions/v2"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func NewTestManifest(prefix string) *v1beta2.Manifest {
+	return &v1beta2.Manifest{
```

Comment:
Maybe establish a ManifestBuilder as well?

Reply:
We can do it later if needed, atm, I do not see the need to have a builder for manifest, manifest doesn't need so many configurations during creation, it's basically converted from module template.